src/desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -69,8 +69,8 @@ module.exports = class ArtworkAuctionView extends Backbone.View
 
     loggedInUser = CurrentUser.orNull()
 
-    # Show the new buy now flow if you have the lab feature enabled
-    if loggedInUser?.hasLabFeature('New Buy Now Flow')
+    # Show the new buy now flow if the feature flag is enabled
+    if ENABLE_NEW_BUY_NOW_FLOW
```

Comment:
I'll point these out, but for some of the cases here it would be really beneficial to include _both_ the lab feature and the feature flag so that we can continue to test the important pieces in production.

So, here `if ENABLE_NEW_BUY_NOW_FLOW || loggedInUser?.hasLabFeature('New Buy Now Flow')`